@here/harp-datasource-protocol/lib/InterpolatedPropertyDefs.ts

```diff
@@ -57,4 +59,16 @@ export interface InterpolatedProperty<T> {
      * Exponent used in interpolation. Only valid with `Exponential` [[InterpolationMode]].
      */
     exponent?: number;
+
+    /**
+     * @hidden
+     * [[StringEncodedNumeral]] type needed to interpret interpolated values back to numbers.
+     */
+    _stringEncodedNumeralType?: StringEncodedNumeral;
+
+    /**
+     * @hidden
+     * Metric unit values array (needed to interpolate mixed meters and pixel values).
```

Comment:
I agree this is not the best name, but I wanted to reflect that this could be used by future supported ``StringEncodedNumeral``s, so that's why I'm not too fond of being so specific...
What do you think of something like ``_dynamicMask``?
(regarding the comment, I'll modify it to clarify).